clients/coredata/value_descriptor.go

```diff
@@ -161,7 +151,15 @@ func (v *valueDescriptorRestClient) ValueDescriptorsUsage(names []string, ctx co
 	q := u.Query()
 	q.Add("names", strings.Join(names, ","))
 	u.RawQuery = q.Encode()
-	data, err := clients.GetRequest(u.String(), ctx)
+
+	// create a new URL client with an empty URL to fulfil the GetRequest contract while relying wholly on
+	// our parsed URL for the actual endpoint data
+	emptyURLClient := urlclient.New(types.EndpointParams{
+		UseRegistry: false,
+		Url:         "",
+	}, nil)
+
+	data, err := clients.GetRequest(u.String(), ctx, emptyURLClient)
```

Comment:
Don't do this.

It looks like you're basically after `GetRequest`'s functionality without the `urlClient.Prefix()` call (and related error checking).  Look at `clients.GetRequest` to extract functionality that you're after into a separate function there.  Call that new function here.  It's a much cleaner and clearer approach.